infrastructure_version.py

```diff
@@ -19,5 +19,5 @@
 # SOFTWARE.
 
 
-OPEN_SOURCE = {'branch': 'master', 'commit_id': '6fceb7b4d7c4f87dddeaca539f1a722c4cb0b623'}
+OPEN_SOURCE = {'branch': 'add_create_file', 'commit_id': 'b3878da306db3850c7f944a47b8a18c8e4c5623c'}
```

Comment:
change to master

Reply:
Now, I can not change to master, because it depends on not merged commit. That is why CI fails